template/Data-Science/Classification/Mobile-User-Behavior-Classification/main.py

```
@@ -0,0 +1,137 @@
+import numpy as np
+from tensorflow.keras.models import load_model
+
+model_folder = './model'
+model_filename = 'user_behavior_model.keras'
+model_path = os.path.join(model_folder, model_filename)
+model = load_model(model_path)
+print(model_path)
+scaler_filename = 'scaler.pkl'
+scaler_path = os.path.join(model_folder, scaler_filename)
+print(scaler_path)
+file = open(scaler_path, 'rb')
```

Ruff (0.8.0)

Use a context manager for opening files

(SIM115)